applications/packages/realtime/common.realtime.sdk-client/src/service/realtime.service.ts

```diff
@@ -3,10 +3,11 @@ import { SocketIOFacade } from "@wraithlight/facade.socketio.client";
 
 export class RealtimeService {
```

Comment:
would be awesome to
* have a sync way to connect with a promise
* have the ability to get notified when
  * connected
  * disconnected